### PR TITLE
Bump for one-off 1.11.0-sf2 release.

### PR DESCRIPTION
**Pull request by klueska**

### CODE REVIEW

**Comment by klueska** [comment]
run all integration tests

**Comment by klueska** [comment]
run all integration tests

**Comment by klueska** [comment]
run all integration tests

**Comment by klueska** [comment]
run all integration tests

File: cli/dcoscli/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # Version is set for releases by our build system.
 # Be extremely careful when modifying.
-version = 'SNAPSHOT'
+version = '1.11.0-sf2'
```
Comment (bamarni) [inline cli/dcoscli/__init__.py]: what does sf2 stands for?

**Pull request closed without merging**